core/domain/exp_domain_test.py

```diff
@@ -5160,7 +5168,11 @@ def test_load_from_v26_textangular(self):
         """Test direct loading from a v26 yaml file."""
         with self.swap(
             html_validation_service, 'get_filename_with_dimensions',
-            mock_get_filename_with_dimensions):
+            mock_get_filename_with_dimensions), self.swap(
+                html_validation_service, 'validate_rte_format',
+                mock_validate_rte_format), self.swap(
+                    html_validation_service, 'validate_customization_args',
+                    mock_validate_customization_args):
```

Comment:
Please use temporary variables to clean up these swaps (here and elsewhere):

```python
mock_get_filename_with_dimensions_context = self.swap(
    html_validation_service, 'get_filename_with_dimensions',
    mock_get_filename_with_dimensions)
mock_validate_rte_format_context = self.swap(...)
mock_validate_customization_args_context = self.swap(...)

with (mock_get_filename_with_dimensions_context,
      mock_validate_rte_format_context,
      mock_validate_customization_args_context):
  # code.
```

Reply:
Done.